test/unit/test_devices/test_ios_device.py

```diff
@@ -1089,7 +1089,7 @@ def test_install_os_install_mode_from_everest_failed(
     mock_set_boot_options.assert_called_with("packages.conf")
     mock_show.assert_called_with(
         f"request platform software package install switch all file {file_system}{image_name} auto-copy",
-        delay_factor=10,
+        delay_factor=20,
     )
     mock_reboot.assert_called()
     mock_os_version.assert_called()
```

Comment:
No unit tests for fastcli?

Reply:
Whole new revamp of fast_cli on the way! 

I also need to add tests for the image booted method that was not present. I'm looking for code examples to test from, and I think I have one from NTC Templates.